README.rst

```diff
@@ -46,3 +46,233 @@ Sample results on macOS with clang and libc++:
 	decimal_from           10555579 ns     10549531 ns           64 items_per_second=94.7909M/s
 	stout_ltoa             40028853 ns     40010824 ns           17 items_per_second=24.9932M/s
 
+
+
+Other benchmarks running on provided by fast_io author. On Linux with libstdc++, GCC-11 C++20.
+
+.. code::
+
+	cqwrteur@DESKTOP-C4VAUFM:~/format-benchmark/build$ ./int-benchmark
+	The number of values by digit count:
+	1  27518
+	2 246950
+	3 450053
+	4 247697
+	5  25016
+	6   2491
+	7    249
+	8     25
+	9      0
+	10      1
+	2020-06-21T08:21:27-04:00
+	Running ./int-benchmark
+	Run on (12 X 3593.26 MHz CPU s)
+	CPU Caches:
+	L1 Data 32 KiB (x6)
+	L1 Instruction 32 KiB (x6)
+	L2 Unified 512 KiB (x6)
+	L3 Unified 16384 KiB (x1)
+	Load Average: 0.31, 0.09, 0.17
+	--------------------------------------------------------------------------------
+	Benchmark                      Time             CPU   Iterations UserCounters...
+	--------------------------------------------------------------------------------
+	sprintf                 66180655 ns     66180815 ns           11 items_per_second=15.1101M/s
+	std_ostringstream       59197433 ns     59196831 ns           12 items_per_second=16.8928M/s
+	std_to_string           14227102 ns     14226740 ns           49 items_per_second=70.2902M/s
+	std_to_chars            10778731 ns     10778754 ns           64 items_per_second=92.7751M/s
+	fmt_to_string           15449511 ns     15449548 ns           46 items_per_second=64.7268M/s
+	fmt_format_runtime      25347564 ns     25347609 ns           28 items_per_second=39.4515M/s
+	fmt_format_compile      15521973 ns     15522015 ns           45 items_per_second=64.4246M/s
+	fmt_format_to_runtime   19493339 ns     19493384 ns           36 items_per_second=51.2995M/s
+	fmt_format_to_compile   11964826 ns     11964854 ns           58 items_per_second=83.5781M/s
+	fmt_format_int           9702564 ns      9702586 ns           72 items_per_second=103.065M/s
+	boost_lexical_cast      29437537 ns     29437599 ns           24 items_per_second=33.9702M/s
+	boost_format           259534333 ns    259534899 ns            3 items_per_second=3.85305M/s
+	boost_karma_generate    10422500 ns     10422529 ns           67 items_per_second=95.946M/s
+	voigt_itostr            16333445 ns     16333491 ns           42 items_per_second=61.2239M/s
+	u2985907                10527758 ns     10527785 ns           66 items_per_second=94.9867M/s
+	decimal_from            11788081 ns     11788109 ns           59 items_per_second=84.8312M/s
+	stout_ltoa              21071739 ns     21071791 ns           33 items_per_second=47.4568M/s
+	fast_io_concat          11085544 ns     11085565 ns           63 items_per_second=90.2074M/s
+	fast_io_print_reserve    9506848 ns      9506866 ns           73 items_per_second=105.187M/s
```

Comment:
This is basically a copy of `fmt_format_int`. You don't need to reimplement the same methods and submit them back =).